src/index.ts

```diff
@@ -142,17 +148,20 @@ const App = async () => {
 
   showStatus('Creating Helia node')
 
+
   const libp2p = libp2pDefaults()
```

Comment:
```suggestion
  const libp2p = libp2pDefaults() as Libp2p<DefaultLibp2pServices>
```

may prevent the need to coerce the Helia type below?

Reply:
This function has a return type, why do we need to cast this at all?

Reply:
In line 158, I had a type assertion because `createHelia` wasn't able to infer the underlying libp2p type (specifically the services) when libp2p was passed in with the default config.

I just changed it from a type assertion to a generic type parameter which feels cleaner.